bigtop-packages/src/charm/sqoop2/layer-sqoop2/tests/01-basic-deployment.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+import amulet
+
+
+class TestDeploy(unittest.TestCase):
+    """
+    Trivial deployment test for Apache Sqoop.
+    """
+    def setUp(self):
+        self.d = amulet.Deployment(series='trusty')
+        self.d.add('sqoop2', 'sqoop2')
+        self.d.setup(timeout=900)
+        self.d.sentry.wait(timeout=1800)
```

Comment:
You need to also setup the java relation, here. Testing on a fresh environment, I stall out, blocked waiting for a java relation.